CMakeLists.txt

```
@@ -12,6 +12,7 @@ option(XTYPES_BUILD_TESTS "Build tests." OFF)
 option(XTYPES_BUILD_EXAMPLES "Build examples." OFF)
 option(XTYPES_EXCEPTIONS "Enable xtypes exceptions in release (which are asserts in debug)." OFF)
 option(XTYPES_BUILD_TOOLS "Build tools." OFF)
+option(THIRDPARTY "Allow to download thidparty repository when needed." ON)
```

Use name mangling to avoid future collisions:
```suggestion
option(XTYPES_THIRDPARTY "Allow to download thidparty repository when needed." ON)
```

---

:+1: 
it was to keep general eProsima CMake option name, but I also think is better with mangling